bartop-ui/src/components/NotFound/__snapshots__/Notfound.test.js.snap

```diff
@@ -2,13 +2,6 @@
 
 exports[`matches the snapshot 1`] = `
 <styled.div>
-  Oops! This page doesn't seem to exist
-   
-  <span
-    aria-label="frowning face"
-    role="img"
-  >
-    ☹️
-  </span>
+  Oops! This page doesn't seem to exist 😓
```

Comment:
Can you use a variable pulled in from your string constants file here? I feel like one of the reasons to use a constants file is to keep everything consistent and changeable only in one place

Reply:
Yep good catch missed this one

Reply:
Oh lol just kidding I didn't miss this, this is the `snapshot` test which is just a tree comparison of what the component renders, like a test for sanity checking, so it has the rendered output. `NotFound` is using `strings.pageNotFound`

Reply:
Oh okay. Still a bummer that you can't render out a variable here so the string isn't defined twice but sounds good

Reply:
No boi it's not being defined twice. `Snapshot` tests take your component and `render` them into a static file, almost like the `HTML` that is output. So this file is basically the rendered markup of the `NotFound` component, it's auto-generated by `jest` none of the jawn is being defined

Reply:
Ohhhhh that is #sick. Cool cool